CDS/scripts/js_utils/youtube/youtube-upload-lib.js

```diff
@@ -131,24 +132,42 @@ function addPosterImageIfExists(connection, videoId, youtubeClient, account) {
 
 function uploadToYoutube(connection) {
     return new Promise((resolve, reject) => {
+        console.log("DEBUG: getting auth client");
```

Comment:
I believe that it's here as the earliest point that something is called - so that we can see that the upload code is _attempting_ to contact the auth code, rather than the auth code is actually running.  In case there is something wrong with the auth code (as indeed there was - at least, that it could not access the security keys)